wscript

```diff
@@ -211,8 +215,18 @@ def configure(conf):
         if conf.check_cc(cflags='-Wall'):
             conf.env.append_value('CFLAGS','-Wall')
     else:
-        if conf.check_cc(cflags='-O2'):
-            conf.env.append_value('CFLAGS','-O2')
+        if not conf.options.optO3:
+            if conf.check_cc(cflags='-O2'):
+                conf.env.append_value('CFLAGS','-O2')
+        else:
+            msg = """\033[91m\033[1mWARNING!
+The configure option \'--O3\' has been specified.
+Use of this option is untested and may result in unvalidated results.
+Press any key to continue or Ctrl+c to abort...\033[0m"""
+            print(msg)
+            input()
```

Comment:
In python2 `input()` could potentially raise an exception. You may want to use `raw_input()` instead: 
```python
try:
    raw_input()
except NameError:
    input()
```